charts/trusted-artifact-signer/templates/segment-backup-job-sa.yaml

```diff
@@ -3,6 +3,6 @@ kind: ServiceAccount
 metadata:
   name: segment-backup-job
   # namespace: {{ .Values.configs.segment_backup_job.namespace }}
-  namespace: sigstore-monitoring
+  namespace: trusted-artifact-signer-monitoring
```

Comment:
This should use the value you set above, currently `configs.tas_monitoring.namespace` but I think it should be `{{ .Values.configs.segment_backup_job.namespace }}`. This is my fault, I never updated it to be changeable.